elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/scaladsl/ElasticsearchFlow.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.elasticsearch.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.elasticsearch.{ElasticsearchFlowStage, ElasticsearchSinkSettings, IncomingMessage}
+import akka.stream.scaladsl.Flow
+import org.elasticsearch.client.{Response, RestClient}
+import spray.json.{DefaultJsonProtocol, JsObject, JsonWriter}
+
+object ElasticsearchFlow {
+
+  /**
```

Comment:
Response of bulk request contains results per command. So it's possible to handle errors if some commands in bulk request failed. I fixed to fail `ElasticsearchFlowStage` when at least one command failed, but I'm not sure this is the best solution.